kerneltools-start

```diff
@@ -73,6 +73,8 @@ for subtool in `echo $subtools | sed -e 's/,/ /g'`; do
             echo "Starting perf record"
             mkdir -p ~/.debug
             cmd="/usr/bin/perf record -a $record_opts"
+            time=$(date -u --date="$(uptime -s)" +%s)
```

Comment:
Don't we need to force this to be UTC time?

Reply:
ie: `TZ=UTC uptime -s`

Reply:
It seems like we do:

Reply:
```
$ uptime -s
2021-11-04 06:38:46

$ TZ=UTC uptime -s
2021-11-04 11:38:46

$ date -u --date="$(TZ=UTC uptime -s)"
Thu Nov  4 11:38:46 UTC 2021

$ date -u --date="$(uptime -s)"
Thu Nov  4 06:38:46 UTC 2021

$ date --date="$(TZ=UTC uptime -s)"
Thu Nov  4 11:38:46 CDT 2021
```

Reply:
Great catch!